taskt/Core/Automation/Commands/RunScriptCommand.cs

```diff
@@ -15,8 +15,8 @@ namespace taskt.Core.Automation.Commands
     public class RunScriptCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Enter the path to the script")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
```

Comment:
You shouldn't remove the variable helper since it provides easy access to argument variables set in the variable manager. You could potentially store the file path in a variable.

Reply:
Gotcha, done.